osu.Game/Overlays/Changelog/ChangelogSupporterPromo.cs

```diff
@@ -138,14 +138,28 @@ private void load(OsuColour colour, TextureStore textures)
                     FillMode = FillMode.Fill,
                     Texture = textures.Get(@"Online/supporter-pippi"),
                 },
-                new Sprite
+                new Container
                 {
                     Anchor = Anchor.TopCentre,
                     Origin = Anchor.TopCentre,
                     Width = 75,
                     Height = 75,
-                    Margin = new MarginPadding { Top = 70 },
-                    Texture = textures.Get(@"Online/supporter-heart"),
+                    Margin = new MarginPadding { Top = 83 },
+                    Masking = true,
+                    EdgeEffect = new EdgeEffectParameters
+                    {
+                        Type = EdgeEffectType.Shadow,
+                        Colour = Color4.HotPink.Opacity(0.88f),
+                        Offset = new Vector2(0, 0),
+                        Radius = 17,
+                        Roundness = 39f,
+                    },
```

Comment:
Where did you get these numbers from? they should probably match how web does it if feasible.

Reply:
For the glow shadow, I found that I wouldn't be able to perfectly replicate it because the EdgeEffect Shadow and the CSS box-shadow are implemented differently, so it couldn't be perfectly relicated without writing an EdgeEffect that replicates CSS's box-shadow.

As for the positioning, I found that pulling the numbers straight from the site's code doesn't really work, (As shown in the promo previously being too far up in lazer when the 70px margin from the site was used). I found that increasing that margin put it more in line with the position of the promo as seen on the site.

Reply:
I think it's fine to change the `Radius` to `10` to match osu-web value.

Also the `Roundness` should be `75 / 2` (container width / 2) to make it perfect circle. You can extract the `75` value as const and reuse it for both heart container and heart sprite size to make those two consistent.